bitnami/airflow/Chart.yaml

```diff
@@ -7,8 +7,8 @@ annotations:
   images: |
     - name: airflow
       image: docker.io/bitnami/airflow:2.10.3-debian-12-r5
-    - name: airflow-exporter
-      image: docker.io/bitnami/airflow-exporter:0.20220314.0-debian-12-r51
+    - name: statsd-exporter
+      image: docker.io/bitnami/airflow-statsd:0.28.0-debian-12-r0
```

Comment:
Shouldn't this be `docker.io/bitnami/statsd-exporter:0.28.0-debian-12-r0`?

Reply:
My fault! Manually edited, fixing it on a new PR